.github/workflows/gitlab-ci.yaml

```
@@ -0,0 +1,86 @@
+image: docker:git
```

actionlint (1.7.4)

unexpected key "image" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)